### ISSUE TITLE
Request: catch `Throwable` to catch all exceptions

### ISSUE THREAD

**Issue by Levivb** [upgate/laravel-jsonrpc#15]
Hey,

See: https://github.com/upgate/laravel-jsonrpc/blob/master/src/Server/Server.php#L146

The `run` method of the `Server` catches `Exception` as final attempt to create a valid response.
Since php 7.0 there's a `Throwable` class which catches even syntax errors and the like.

Laravel itself does catch all exceptions via `} catch (Throwable $e) {`
See https://github.com/laravel/framework/blob/5.8/src/Illuminate/Routing/Pipeline.php#L33

Would it be possible for this package to do the same?
Example code snippet:
```php
        } catch (JsonRpcException $e) {
            return $request->getId() ? RequestResponse::constructExceptionErrorResponse($request->getId(), $e) : null;
        } catch (\Throwable $e) {
            return $this->handleException($e, $request);
        }
```

Let me know what you think :)

**Comment by kbaryshnikov** [comment]
Please see if #16 works for you.

**Comment by Levivb** [comment]
Thanks for the fast response! 👍

This works perfectly

The `} catch (\Exception $e) {` in the method `public function execute(RequestInterface $request)` of `src/Server/Server.php` could also be changed to `} catch (\Throwable $e) {`

**Comment by kbaryshnikov** [comment]
Yeah that makes sense, thanks! Updated the PR.

**Comment by Levivb** [comment]
Cool! That goes a long way in providing a stable and consistent JSON RPC API 👍

**Comment by kbaryshnikov** [comment]
Merged #16 to master.

**Comment by kbaryshnikov** [comment]
https://github.com/upgate/laravel-jsonrpc/releases/tag/0.5.0
